### ISSUE TITLE
New Rule: computed-property-style

### ISSUE THREAD

**Issue by Alonski** [ember-cli/eslint-plugin-ember#344]
I just got some help from @rwjblue building this codemod:
https://github.com/Alonski/ember-computed-getter-codemod
Might be useful to turn it into an ESLint rule that is fixable.

Most basic usage of this rule is this:

Good:
```js
Ember.Object.extend({
    myComputed: computed({
        get() {
            return "myComputed";
        }
    })
});
```
Bad:
```js
Ember.Object.extend({
    myComputed: computed(function() {
        return "myComputed";
    })
});
```

With computed args:
Good:
```js
Ember.Object.extend({
    myComputed: computed("computifyThis", {
        get() {
            return "myComputed";
        }
    })
});
```
Bad:
```js
Ember.Object.extend({
    myComputed: computed("computifyThis", function() {
        return "myComputed";
    })
});
```

**Comment by Alonski** [comment]
After chatting on Discord https://discordapp.com/channels/480462759797063690/486516209873846292/516889258146004993

It was decided that the default would be the opposite of what is written above. However there will be configuration for this to flip the logic if someone prefers it the other way.

Bad:
```
Ember.Object.extend({
    myComputed: computed({
        get() {
            return "myComputed";
        }
    })
});
```
Good:
```
Ember.Object.extend({
    myComputed: computed(function() {
        return "myComputed";
    })
});
```

**Comment by jrjohnson** [comment]
@Alonski I was just going to figure out how to enforce this in our app. Did you make any progress? If not I'll get started on a PR for a rule.

**Comment by Alonski** [comment]
@jrjohnson I havent put any time into this yet. If you can that would be awesome. Check out the codemod for inspiration

**Comment by lukasMega** [comment]
Hello guys, what about consider old style:

Bad:
```
Ember.Object.extend({
    myComputed: function() {
        return "myComputed";
    }.property('a', 'b')
});
```
Good:
```
Ember.Object.extend({
    myComputed: computed('a', 'b', function() {
        return "myComputed";
    })
});
```

I think there are people that would consider it useful, including me. Maybe for this we should have a separate rule with name like `computed-property-style-old`.
What do you think?
